deploy/charts/monitoring/values.yaml

```diff
@@ -21,3 +21,22 @@ persistence:
   storageClass: null
 
 nodeSelector: {}
+
+kube-state-metrics:
+  releaseNamespace: true
+
+alertmanager:
+  persistence:
+    enabled: false
+  config:
+    receivers:
+      - name: pagerduty-notifications
+        pagerduty_configs:
+          - service_key: ${PAGERDUTY_INTEGRATION_KEY}
+    route:
+      receiver: pagerduty-notifications
+      group_by: [alertname]
+  configmapReload:
+    enabled: true
```

Comment:
Looks like this shouldn't be here as it wasn't in https://github.com/stackrox/stackrox/pull/4655/files#diff-28bf2ebb1c3d4a5a12067bcf8c7ee3ad72940e8660d903e74ebe9cbc175c9a36

```suggestion
```